### PR TITLE
Remove parameter `normalize` in linear models

### LINKED ISSUE #660: Parameter `normalize` in linear models
### 🐛 Bug Report

FutureWarning: 'normalize' was deprecated in version 1.0 and will be removed in 1.2. Please leave the normalize parameter to its default value to silence this warning. The default behavior of this estimator is to not do any normalization. If normalization is needed please use sklearn.preprocessing.StandardScaler instead.

### Expected behavior

I guess we need to remove the parameter `normalize` from the constructor of linear models

### How To Reproduce

```
from etna.datasets import TSDataset
from etna.pipeline import Pipeline
from etna.models import LinearPerSegmentModel
import pandas as pd
from etna.transforms import SegmentEncoderTransform

df = pd.DataFrame({"segment":"a",
                   "target":[1,2,3,4,5,6,7],
                   "timestamp": pd.date_range("2000-01-01", "2000-01-07")})
ts = TSDataset(df=TSDataset.to_dataset(df), freq="D")
ts.fit_transform([SegmentEncoderTransform()])
model = LinearPerSegmentModel()
model.fit(ts)
```

### Environment

_No response_

### Additional context

_No response_

### Checklist

- [X] Bug appears at the latest library version

### PR DESCRIPTION
**Pull request by Mr-Geekman**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [x] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Proposed Changes
Look #660.

## Related Issue
#660.

## Closing issues
Closes #660.

### CODE REVIEW

**Review by martins0n — APPROVED** [review]
👍
